plugin_gems.rb

```diff
@@ -19,3 +19,4 @@
 download "fluent-plugin-rewrite-tag-filter", "1.5.5"
 download "fluent-plugin-td-monitoring", "0.2.2"
 download "fluent-plugin-google-cloud", "0.5.2"
+download "fluent-plugin-detect-exceptions", "0.0.1"
```

Comment:
Bumped to the newest version 0.0.3 that is about to be submitted to Rubygems.org.